performance_test/CMakeLists.txt

```diff
@@ -167,8 +206,18 @@ if(PERFORMANCE_TEST_CYCLONEDDS_ENABLED)
     )
 endif()
 
+if(ODB_FOR_SQL_ENABLED)
+    include_directories(${CMAKE_CURRENT_BINARY_DIR}
+        ${CMAKE_CURRENT_SOURCE_DIR}/src/experiment_configuration
+        ${CMAKE_CURRENT_SOURCE_DIR}/src/experiment_execution
+        ${ODB_INCLUDE_DIRS})
+endif()
+
 set(EXE_NAME perf_test)
-add_executable(${EXE_NAME} ${sources})
+add_definitions(-DEXE_NAME="${EXE_NAME}")
```

Comment:
Why do we need this? And if we would need it we need to prefix it as we did with all other defines to avoid conflicts with 3rd party symbols.

Reply:
We don't need this anymore, removing this line.